media/js/base/mozilla-global-nav.js

```diff
@@ -0,0 +1,273 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var _closeButton = document.getElementById('nav-drawer-close-button');
```

Comment:
Note: the reason I'm using plain JS here is with the assumption that this will eventually be used on more than just mozorg, so I'm trying to avoid too many dependencies.